server/sonar-server/src/main/java/org/sonar/server/projectanalysis/ws/CreateEventAction.java

```diff
@@ -104,6 +104,8 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request httpRequest, Response httpResponse) throws Exception {
+    userSession.checkLoggedIn();
```

Comment:
Also, I don't see why this change of behavior is requiered. "Anyone" can be given admin permission on a project.